src/configure.py

```diff
@@ -202,6 +204,19 @@ def onmindelegation(input):
     fsm.go()
 
 
+def onminpayment(input):
+    try:
+        if not input:
+            input = "0"
+        global parser
+        parser.set(MIN_PAYMENT_AMT, float(input))
+        parser.validate_service_fee(parser.get_conf_obj())
```

Comment:
Why validate service fee here? And is the text for the exception correct?

Reply:
This is dumb copy-paste from `onmindelegation` which seems to be the same dumb copy-paste from `onservicefee`.
Fixed both `onmindelegation` and `onminpayment`.